bootstrap.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+sudo yum -y install pam-devel
+sudo yum -y install readline*
+sudo yum -y install openssl*
+
```

Comment:
Need some additional packages:

- [ ] libmemcached-devel
- [ ] libedit-devel
- [ ] I had to replace /lib64/libldap_r-2.4.so.2 with the PPAS version, as described here: https://groups.google.com/forum/#!topic/repmgr/TS7QfYEoNoY